src/QsCompiler/Compiler/Templates/CppInterop.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint;
+
+namespace Microsoft.Quantum.QsCompiler.Templates
+{
+    public class ArgumentCpp : Argument
```

Comment:
We should make this class `internal` since we don't want to expose it outside the binary.

Reply:
I think this still needs to be changed to `internal`.